admin/sql/CreateTables.sql

```diff
@@ -847,6 +847,13 @@ CREATE TABLE event_type ( -- replicate
     gid                 uuid NOT NULL
 );
 
+CREATE TABLE first_recording_release_date (
```

Comment:
`recording_first_release_date` would better fit the current naming scheme where _`entity_*`_ tables relate to _entity_ type; It would also better match the rest of your code in this pull request that refer to `*first_release_date*` only.

Reply:
Good point! I'll working on updating that soon.